src/js/view/controls/info-overlay.js

```diff
@@ -57,7 +68,10 @@ export default function (container, model, api, onVisibility) {
             }
             durationContainer.textContent = durationText;
         }, instance);
-        clientIdContainer.textContent = `Client ID: ${getClientId()}`;
+
+        if (clientIdContainer) {
```

Comment:
what is the benefit to showing/hiding the div vs. making `textContent = ''` here? are theree styling implications to leaving the div there but not having any info inside of it?

Reply:
I figured we wouldn't want to create the div to begin with if client id isn't available

Reply:
ah fair!

Reply:
ah totally!

Reply:
ahh fair!

Reply:
Joe's suggestion would require less code as the template would remain unchanged (smaller) and you would call `shouldTrackUser()` here (lazier) rather than up top.
`clientIdContainer.textContent = shouldTrackUser() ? '' : `Client ID: ${getClientId()}`;`